doc/reference/configuration/configuration_reference.rst

```diff
@@ -3599,6 +3599,165 @@ The ``replication`` section defines configuration parameters related to :ref:`re
+        ``prefix: '{{ replicaset_name }}'``
+
+      - For instances matching a specific pattern (e.g., `i-\d\d\d`), set:
+        ``prefix: 'i-'``
+
+    **Example**
+
+    1. Create a ``config.yaml`` file with the following content:
+
+    .. code-block:: yaml
+
+        credentials:
+          users:
+            guest:
+              roles: [super]
+
+        replication:
+          failover: manual
+          autoexpel:
+            enabled: true
+            by: prefix
+            prefix: '{{ replicaset_name }}'
+
+        iproto:
+          listen:
+            - uri: 'unix/:./var/run/{{ instance_name }}.iproto'
+
+        groups:
+          g-001:
+            replicasets:
+              r-001:
+                leader: r-001-i-001
+                instances:
+                  r-001-i-001: {}
+                  r-001-i-002: {}
+                  r-001-i-003: {}
+
+
+    This configuration:
+	- Sets up authentication with a guest user assigned the super role.
+	- Enables the ``autoexpel`` option to automatically expel instances not present in the YAML file.
+	- Defines instance names based on a prefix pattern: ``{{ replicaset_name }}``.
+	- Lists three instances: ``r-001-i-001``, ``r-001-i-002``, and ``r-001-i-003``.
+
+
+    2. Open terminal window and start three instances using the following commands:
+
+    .. code-block:: lua
+
+        tarantool --name r-001-i-001 --config config.yaml -i
+
+
+    .. code-block:: lua
+
+        tarantool --name r-001-i-002 --config config.yaml -i``
+
+
+    .. code-block:: lua
+
+        tarantool --name r-001-i-003 --config config.yaml -i
+
+    3. Edit ``config.yaml`` and remove the following entry for ``r-001-i-003``:
+
+    .. code-block:: lua
+        r-001-i-003: {}
+
+
+    The updated ``config.yaml`` should look like this:
+
+    .. code-block:: yaml
+
+        groups:
+          g-001:
+            replicasets:
+              r-001:
+                leader: r-001-i-001
+                instances:
+                  r-001-i-001: {}
+                  r-001-i-002: {}
+
+    Save the file.
+
+    4. For the leader instance (``r-001-i-001``), check the ``_cluster`` space:
+
+    .. hint::
+
+        The ``_cluster`` system space in Tarantool stores metadata about all instances currently recognized as part of the cluster.
+        It shows which instances are registered and active.
+
+    You should see ``r-001-i-003`` still listed in the ``_cluster`` system space.
+
+    5. Reload the configuration:
+
+    .. code-block:: lua
+
+        config = require('config')
+            config:reload()
+
+    6. Verify the changes:
+
+    .. code-block:: lua
+
+        box.space._cluster:fselect()
+
+    After the reload, ``r-001-i-003`` should no longer appear in the ``_cluster`` system space.
+
+    |
+    | Type: boolean
+    | Default: false
+    | Environment variable:
+    | TT_REPLICATION_AUTOEXPEL_BY
+    | TT_REPLICATION_AUTOEXPEL_ENABLED
+    | TT_REPLICATION_AUTOEXPEL_PREFIX
```

Comment:
It needs some update. At least `Default: false` is related to `autoexpel.by`, but not other options.

Reply:
yep, I'll rewrite as we discussed. I mean 3 subsectiond for 3 env var